Dockerfile

```diff
@@ -15,7 +15,13 @@ RUN curl -sSL https://dl.google.com/linux/linux_signing_key.pub | apt-key add -
 
 RUN apt-get update && apt-get install -y python-pip
 
-RUN pip install awscli
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" \
+&& unzip awscliv2.zip \
+&& ./aws/install
+
+RUN aws s3 cp s3://kibana.bfs.vendor/aes/chrome/google-chrome-stable_79.0.3945.117-1_amd64.deb ./tmp/google-chrome.deb \
```

Comment:
If this is added, do you still need line 14 to install the chrome 94?

Reply:
Thanks for catching that! New commit removes that line with a description.